pkg/rules/manager.go

```diff
@@ -135,6 +136,10 @@ func NewManager(
 		extLset:   extLset,
 		ruleFiles: make(map[string]string),
 	}
+
+	span, ctx := tracing.StartSpan(ctx, "new_manager")
+	defer span.Finish()
```

Comment:
I think this is not needed.

Reply:
Thanks for the review!
I've finished fixing it.